### PR TITLE
roll random was always one less than possible sides

### PR DESCRIPTION
**Pull request by SpencerSharkey**
Make roll fair!

### CODE REVIEW

**Comment by meepdarknessmeep** [comment]
.roll 10d0

**Comment by meepdarknessmeep** [comment]
:-1:

**Comment by meepdarknessmeep** [comment]
add one to the result, don't add one to the arg

**Comment by meepdarknessmeep** [comment]
spam

**Comment by SpencerSharkey** [comment]
no, if you did roll 10d2 you'd only roll 1

**Comment by meepdarknessmeep** [comment]
Meepen Ƹ̵̡Ӝ̵̨̄Ʒ: .roll 10d0
$: Meepen Ƹ̵̡Ӝ̵̨̄Ʒ rolls 10 d0s: 0, 0, 0, 0, 0, 0, 0, 0, 0, 0

**Comment by SpencerSharkey** [comment]
there :shipit:

File: plugins/roll.js [hunk 1]
```diff
@@ -1,12 +1,7 @@
-// Returns a random integer between 1 and max
-function randInt( max ) {
-	return Math.floor( Math.random() * ( max - 1 ) ) + 1;
-}
-
 bot.registerCommand( "roll", function( name, _, _, arg_str, group ) {
 
 	if ( /(\d+)?d\d+/.test( arg_str ) ) {
-
+
```
Comment (zerfgog) [inline plugins/roll.js]: :point_right: :point_down:
:point_up: :point_left:

Comment (SwadicalRag) [inline plugins/roll.js]: spin

**Comment by zerfgog** [comment]
I would say +1 but in this case it's more of a -1!!!!!!!!

**Comment by SpencerSharkey** [comment]
merge

**Comment by SwadicalRag** [comment]
you have not appeased the Merge gods yet

**Comment by SpencerSharkey** [comment]
ok no more indentation

File: plugins/roll.js [hunk 2]
```diff
@@ -24,7 +18,6 @@ bot.registerCommand( "roll", function( name, _, _, arg_str, group ) {
 		msg += ": " + results.join( ", " );
 
 		bot.sendMessage( msg, group );
-
 	} else {
 		// Just roll d20
 		bot.sendMessage( name + " rolls a d20: " + randInt( 20 ), group )
```
Comment (mcd1992) [inline plugins/roll.js]: You're still using randInt here. Fix it nurd.

**Pull request merged**